host-bmc/host_pdr_handler.cpp

```diff
@@ -544,11 +544,22 @@ void HostPDRHandler::mergeEntityAssociations(
                 }
             }
 
-            // Record Handle is 0xFFFFFFFF(max value uint32_t), for merging
-            // entity association pdr to bmc range
-            pldm_entity_association_pdr_add_from_node(
-                node, repo, &entities, numEntities, true,
-                isHostUp() ? TERMINUS_HANDLE : terminus_handle, 0xFFFFFFFF);
+            if (isHostUp() || (terminus_handle & 0x8000))
```

Comment:
Adding a comment to explain why the condition (terminus_handle & 0x8000) is added will help others

Reply:
done